lesson-5/Dooley - Lesson 5/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Sample Project</title>
+    <link rel="stylesheet" href="style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Work+Sans:400,700" rel="stylesheet">
+  </head>
+
+  <body>
+	<div class="container">
+	  	<img src="images/sample-project.jpg" width="300px">
+  	
+    	<div class="text">
```

Comment:
This whole block should be indented one step to the right since it is within `.container`